services/blog.py

```
@@ -389,3 +394,172 @@ def _generate_content(self, digest: Dict[str, Any]) -> str:
+    def generate_ai_blog(self, target_date: str) -> Dict[str, Any]:
+        """
+        Generate an AI-written blog draft from a digest.
+        
+        Args:
+            target_date: Date in YYYY-MM-DD format
+            
+        Returns:
+            Dictionary containing the AI-generated blog with frontmatter and body
+        """
+        if not self.worker_url:
+            raise ValueError("CLOUDFLARE_WORKER_URL not configured in environment")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)